Gems/AtomContent/Sponza/Assets/objects/sponza_mat_ceiling.material

```diff
@@ -16,7 +16,8 @@
             ]
         },
         "irradiance": {
-            "color": [
+            "irradianceColorSource": "Manual",
```

Comment:
Hm, why are some of these files still showing property values specified in a tree? I made changes so that new saved .material files would have a flat list (see 103_DetailMaps_BaseColor.material for example).

If you updated this file by opening and saving in Material Editor, I would expect the property values to be specified like:
```
    "irradiance.irradianceColorSource": "Manual",
```

Reply:
I edited the material files with a text editor. Going through an open-save cycle in the Material Editor does indeed flatten the properties (so everything is working fine there).